src/walkway.js

```diff
@@ -321,7 +321,11 @@
   function Path(path, duration, easing) {
     WalkwayElement.call(this, path, duration, easing);
 
-    this.length = path.getTotalLength();
+    var svg = path.closest('svg');
```

Comment:
Perhaps we could pull this out into a function -- `getElementScale` -- and put the same code in the Line and Polyline constructors. What do you think?